### PR TITLE
Issue with hasOne dropdowns

### PR DESCRIPTION
**Pull request by DarkSide666**
In progress...

### CODE REVIEW

File: demos/init-db.php [hunk 1]
```diff
@@ -181,7 +183,7 @@ protected function init(): void
     public function importFromFilesystem($path, $isSub = false)
     {
         if (!$isSub) {
-            $path = __DIR__ . '/' . $path;
+            $path = __DIR__ . '/../' . $path;
```
Comment (mvorisek) [inline demos/init-db.php:186]: I think this is solely for demos...

**Comment by mvorisek** [comment]
@DarkSide666 please finish if there is an issues

**Comment by DarkSide666** [comment]
Closed in favor of https://github.com/atk4/ui/commit/9f21074a2d6eb680d5339a4136606440b3641894

And yes I know - accidentally pushed to develop. Will fix if needed.

**Comment by mvorisek** [comment]
Is it tough to disable push to protected branches even for admins?

**Pull request closed without merging**